nipoppy/trackers/run_tracker.py

```diff
@@ -161,19 +161,23 @@ def run(global_configs, dash_schema_file, pipelines, session_id="ALL", run_id="1
 
                 if pipeline == "heudiconv":
                     subject_dir = f"{DATASET_ROOT}/bids/{bids_id}"
```

Comment:
Would it be better to only have `subject_ses_dir` (i.e. no `subject_dir`) and just pass `subject_ses_dir` to the trackers? Then all the trackers would have to be modified but it might be cleaner in the end.

Reply:
I was thinking the same, but we have to pass the `session_id` to pipelines with BIDS naming to construct the expected filename. So not sure if it would help much.